lib/components/viewers/live-stop-times.tsx

```diff
@@ -191,14 +202,18 @@ class LiveStopTimes extends Component<Props, State> {
             <FormattedMessage id="components.LiveStopTimes.autoRefresh" />
           </label>
           <button
-            // Functionality is provided by auto-refresh, this only adds confusion
-            // in a screen reader context
-            aria-hidden
+            aria-label={refreshButtonText}
             className="link-button pull-right percy-hide"
             onClick={this._refreshStopTimes}
             style={{ fontSize: 'small' }}
+            title={refreshButtonText}
```

Comment:
Do we need both title and aria label?

Reply:
Oh I thought `title` was automatically read out by screen readers. Thanks for clearing that up!